others/binary_to_decimal.cpp

```diff
@@ -0,0 +1,89 @@
+/**
+ * @file
+ * @brief [Binary to Decimal conversion]
+ * @details
+ * Given a binary number (string), convert it to decimal format.
+ * Time Complexity: O(n)
+ * Space Complexity: O(n)
+ *
+ * @author [Vishwa Krishnan](https://github.com/vishwakrishnan02)
+ */
+
+#include <cassert>
+#include <iostream>
+#include <string>
+#include <algorithm>
```

Comment:
Please add a one-line description of what the library/header is for (**see the example below**).

```c++
#include <iostream>    /// for IO operations
#include <cassert>    /// for assert
```